.github/workflows/integration-test.yaml

```diff
@@ -0,0 +1,292 @@
+       GOPATH=$(go env GOPATH) make clean && make -j$nproc IMAGES='ceph' VERSION=0 build
+       docker images
+       docker tag $(docker images|awk '/build-/ {print $1}') rook/ceph:master
+
+    - name: TestCephHelmSuite
+      run: |
+       tests/scripts/minikube.sh helm
+       tests/scripts/helm.sh up
+       sed -i "s|deviceFilter:  ''|deviceFilter: $(lsblk|awk '/14G/ {print $1}'| head -1)|g" tests/framework/installer/ceph_manifests.go
+       go test -v -timeout 1800s -run CephHelmSuite github.com/rook/rook/tests/integration
+
+    - name: The job has failed
+      if: ${{ failure() }}
+      run: | 
+       cd /home/runner/work/rook/rook/tests/integration/_output/tests/
+       ls
+       FILE=$(find . ! -name '*_previous_*' -name 'TestCephHelmSuite_localhost_helm-ns_rook-ceph-operator-*.log')
+       test -f $FILE && cat $FILE
+       FILE_PREVIOUS=$(find . -name '*_previous_*' -name 'TestCephHelmSuite_localhost_helm-ns_rook-ceph-operator-*.log')
+       test -f $FILE_PREVIOUS && cat $FILE_PREVIOUS         
+      
+  TestCephMultiClusterDeploySuite:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+
+    - name: setup golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
+
+    - name: setup minikube
+      uses: manusa/actions-setup-minikube@v2.0.1
+      with:
+        minikube version: 'v1.13.1'
+        kubernetes version: 'v1.19.2'
+        driver: docker
+        start args: --memory 6g --cpus=2
+
+    - name: check k8s cluster status
+      run: |
+        kubectl cluster-info
+        kubectl get pods -n kube-system
+
+    - name: use local disk
+      run: |
+        sudo swapoff --all --verbose
+        sudo umount /mnt
+        # search for the device since it keeps changing between sda and sdb
+        sudo wipefs --all --force /dev/$(lsblk|awk '/14G/ {print $1}'| head -1)1
+        sudo lsblk
+
+    - name: build rook
+      run: |
+       # set VERSION to a dummy value since Jenkins normally sets it for us. Do this to make Helm happy and not fail with "Error: Invalid Semantic Version"
+       GOPATH=$(go env GOPATH) make clean && make -j$nproc IMAGES='ceph' VERSION=0 build
+       docker images
+       docker tag $(docker images|awk '/build-/ {print $1}') rook/ceph:master
+
+    - name: TestCephMultiClusterDeploySuite
+      run: |
+       export TEST_SCRATCH_DEVICE=$(sudo lsblk --paths|awk '/14G/ {print $1}'| head -1)1
+       sed -i "s|deviceFilter:  ''|deviceFilter: $(lsblk|awk '/14G/ {print $1}'| head -1)|g" tests/framework/installer/ceph_manifests.go
+       go test -v -timeout 1800s -run CephMultiClusterDeploySuite github.com/rook/rook/tests/integration
+
+    - name: The job has failed
+      if: ${{ failure() }}
+      run: | 
+       cd /home/runner/work/rook/rook/tests/integration/_output/tests/
+       ls
+       FILE=$(find . ! -name '*_previous_*' -name 'TestCephMultiClusterDeploySuite_localhost_mrc-n1-system_rook-ceph-operator-*.log')
```

Comment:
We should probably write a collector script at some point.

Reply:
Agreed. I'll do it once all the tests will pass.